docs/core-packages/traverse.md

```diff
@@ -0,0 +1,9 @@
+---
+layout: docs
+title: babel-traverse
+description: The babel-traverse module maintains the overall tree state, and is responsible for replacing, removing, and adding nodes.
+permalink: /docs/core-packages/traverse/
+package: babel-traverse
+---
+
+{% include package_readme.html %}
```

Comment:
There is no README for babel-traverse, the page is empty on the website.

I made this https://gist.github.com/xtuc/26931c424fb7148c000217de8e7fdf76 some time ago. We could publish this.